aleph-client/src/lib.rs

```diff
@@ -172,6 +181,18 @@ pub trait AnyConnection: Clone + Send {
     ) -> T {
         self.read_constant_or_else(pallet, constant, Default::default)
     }
+
+    fn read_storage_map<K: Encode, T: Decode + Clone>(
+        &self,
+        pallet: &'static str,
+        map_name: &'static str,
+        map_key: K,
+        block_hash: Option<H256>,
+    ) -> Option<T> {
```

Comment:
why don't we unwrap it once again? all the old methods in this trait were supposed to return double-unwrapped values

the same question goes to `read_storage_value_from_block`

Reply:
I can do that, no problem. I am just worrying a bit that some of those calls kind of lie - they treat `None` as decode error, but it only means `everything fine, just returned null`.